Engine/Source/MeshRendererComponent.cpp

```diff
@@ -139,7 +133,7 @@ void MeshRendererComponent::Load(unsigned int meshUid, unsigned int materialUid)
 void MeshRendererComponent::Update()
 {
 	mOBB.SetFrom(mAABB, mOwner->GetWorldTransform());
-	Draw();
+	//Draw();
```

Comment:
This is where the frustum culling is performed in the draw function, in this pull request you kill the culling and there is no time now to modify the culling into the Render list